packages/cookie-consent/src/CookieConsentProvider/types.ts

```diff
@@ -0,0 +1,20 @@
+export type CategoryKind =
+  | 'essential'
+  | 'functional'
+  | 'marketing'
+  | 'analytics'
+  | 'advertising'
+
+export type Consent = { [K in CategoryKind]: boolean }
+
+type Integration = { category: CategoryKind; name: string }
+
+export type Integrations = Integration[]
+
+// TODO: avoid duplicating this type in @shire/console
+export type Config = {
+  segment?: {
+    writeKey: string
+    cdnURL: string
+  } | null
+}
```

Comment:
Can be passed as props instead of mimicking the console config object :)

Also no need to mention the console project, should be fully unrelated